src/OpenTelemetry/Trace/TracerProviderBuilderExtensions.cs

```diff
@@ -71,41 +71,31 @@ public static TracerProviderBuilder AddProcessor(this TracerProviderBuilder trac
             return tracerProviderBuilder;
         }
 
-        public static TracerProvider Build(this TracerProviderBuilder tracerProviderBuilder)
-        {
-            if (tracerProviderBuilder is TracerProviderBuilderSdk tracerProviderBuilderSdk)
-            {
-                return tracerProviderBuilderSdk.Build();
-            }
-
-            return null;
-        }
-
         /// <summary>
-        /// Adds a DiagnosticSource based instrumentation.
-        /// This is required for libraries which is already instrumented with
-        /// DiagnosticSource and Activity, without using ActivitySource.
+        /// Adds an activity listener for activities created the legacy way with the provided OperationName. The listener samples and processes these activities
```

Comment:
/// Adds activity with given operation name to the list of subscribed activities. This is for legacy activities (i.e activities created without using ActivituSource API).

Reply:
I have updated the description.